keycloak/themes/hmda/login/messages/messages_en.properties

```diff
@@ -1,21 +1,21 @@
-missingInstitutionMessage=&#8226; Please specify the Institution(s) for which you will be filing HMDA data.
-invalidInstitutionMessage=&#8226; Institution <strong>{0}</strong> is not associated with <strong>{1}</strong> email domain.
+missingInstitutionMessage=&#8226; Please specify the financial institution(s) for which you will be filing HMDA data.
+invalidInstitutionMessage=&#8226; The financial institution <strong>{0}</strong> is not associated with <strong>{1}</strong> email domain.
 unknownEmailDomainMessage=&#8226; The email domain <strong>{0}</strong> is not in the known list of HMDA filer domains.
-institutionErrorMessage=&#8226; An error occurred while validating the institution(s) against the <strong>{0}</strong> email domain.
-missingFirstNameMessage=&#8226; Please specify first name.
-missingLastNameMessage=&#8226; Please specify last name.
-missingEmailMessage=&#8226; Please specify email.
-missingUsernameMessage=&#8226; Please specify username.
-missingPasswordMessage=&#8226; Please specify password.
-missingTotpMessage=&#8226; Please specify authenticator code.
+institutionErrorMessage=&#8226; An error occurred while validating the financial institution(s) against the <strong>{0}</strong> email domain.
+missingFirstNameMessage=&#8226; Please specify the user''s first name.
```

Comment:
I updated the `''` in https://github.com/cfpb/hmda-platform-auth/pull/83/commits/bae16484c9d11aca2af721521040c934a4bdf25a but do you think these would make more sense as `Please specify your first name.`?